crates/epoch-snark/src/gadgets/epochs.rs

```diff
@@ -98,14 +100,14 @@ impl ConstraintSynthesizer<Fr> for ValidatorSetUpdate<Bls12_377> {
 }
 
 impl ValidatorSetUpdate<Bls12_377> {
-    fn enforce<CS: ConstraintSystem<Fr>>(&self, cs: &mut CS) -> Result<EpochBits, SynthesisError> {
+    fn enforce(&self, cs: ConstraintSystemRef<<Bls12_377_Parameters as Bls12Parameters>::Fp>) -> Result<EpochBits, SynthesisError> {
```

Comment:
Is a nicer style having the `cs` as a field?

Reply:
Seems unnecessary given it's only used once as a function parameter. Both styles are probably identical in terms of functionality but type inference makes me slightly nervous about declaring it first as a type

Reply:
Got it. Makes sense to me to leave as is.